jobservatory_react/src/components/detail/ResponsiveLineComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import { useColorMode } from "@chakra-ui/react";
+import { ResponsiveLine } from "@nivo/line";
+import React from "react";
+import { ChartLine } from "./Detail";
+
+interface DetailChartProps {
+  formattedChartData: ChartLine[];
+}
+
+export const ResponsiveLineComponent = ({
+  formattedChartData,
+}: DetailChartProps) => {
+  console.log("formattedChartData :>> ", formattedChartData);
+  for (let i = 0; i < formattedChartData.length; i++) {
+    const dataDatesArray = formattedChartData[i];
+    dataDatesArray.data.sort(
```

Comment:
Igual podemos cachar si podemos hacer la query de mongodb ordenada por fecha?

Reply:
siiii po papá, PR open with that 👍